common/src/main/java/org/corfudb/common/util/URLUtils.java

```diff
@@ -0,0 +1,134 @@
+package org.corfudb.common.util;
+
+import io.netty.channel.ChannelHandlerContext;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.conn.util.InetAddressUtils;
+
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.UnknownHostException;
+
+/**
+ * Contains Utility methods to work with URIs or the Socket Addresses.
+ * Supports both IPv4 and IPv6 methods.
+ *
+ * Created by cgudisagar on 1/22/23.
+ */
+@Slf4j
+public final class URLUtils {
+
+    private static final String COLON_SEPERATOR = ":";
+
+    private URLUtils() {
+        // prevent instantiation of this class
+    }
+
+    public static String getVersionFormattedEndpointURL(String host, int port) {
+        return getVersionFormattedEndpointURL(host, Integer.toString(port));
+    }
+
+    public static String getVersionFormattedEndpointURL(String host, Integer port) {
```

Comment:
Integer and int are auto-convertible through autoboxing and unboxing.  Separate methods for each are not needed IMO

Reply:
Done, Thank you.